src/Views/scstormrunoff.html

```diff
@@ -32,13 +33,63 @@ <h4 class="modal-title hidden-print">Study Area Storm Event Runoff Summary</h4>
                                     <button style="line-height:unset; display:inline-block; width:49%;" class="btn-wim-sm btn-wim-success btn-red" ng-click="vm.clearResults(BohmanRuralForm)" ng-hide="!vm.validateForm(BohmanRuralForm)"><span class="fa fa-trash-o"></span>&nbsp;&nbsp;Clear All</button>
-                                    <button style="line-height:unset; display:inline-block; width:49%;" class="btn-wim-sm btn-wim-success btn-wim-group-1 btn-blue" ng-click="vm.queryRegressionRegions()" ng-hide="!vm.validateForm(BohmanRuralForm)"><span ng-class="vm.canContinue ? 'fa fa-check-circle' : 'fa fa-refresh fa-spin'"></span>{{ vm.showResults ? '&nbsp;&nbsp;Recalculate Results' : '&nbsp;&nbsp;Continue' }}</button>
+                                    <button style="line-height:unset; display:inline-block; width:49%;" class="btn-wim-sm btn-wim-success btn-wim-group-1 btn-blue" ng-click="vm.queryRegressionRegions('Rural')" ng-hide="!vm.validateForm(BohmanRuralForm)"><span ng-class="vm.canContinue ? 'fa fa-check-circle' : 'fa fa-refresh fa-spin'"></span>{{ vm.showRuralResults ? '&nbsp;&nbsp;Recalculate Results' : '&nbsp;&nbsp;Continue' }}</button>
                                 </div>
                                 <label class="fullwidth alignc orange" ng-if="!vm.validateForm(BohmanRuralForm)"><span class="fa fa-exclamation-triangle"></span>&nbsp;&nbsp;Parameters Must Be Entered Before Results Can Be Calculated</label>
                                 <label ng-if="vm.validateForm(BohmanRuralForm)">&nbsp;&nbsp;</label> 
-                            </form>  -->
-                            <div> Coming Soon. </div>
-                            <br>
+                            </form>
+                            <div ng-if="vm.showRuralResults">
+                                <div style="position:relative;">
+                                    <p class="fullwidth alignc hidden-print" style="font-size: 16px;"><b>USGS SC Flood Hydrograph for Rural Watersheds using {{vm.SelectedAEP.name}} AEP</b></p>
+                                    <div class="wim-alert-warning">{{vm.warningMessages}}</div>
+                                    <div class="well">
+                                        <h3>Parameters</h3>
+                                        <table id="BohmanRuralParameterTable" class="table table-condensed smalltext">
+                                            <tbody>
+                                                <tr>
+                                                    <td>Clicked Point (Latitude, Longitude):</td>
+                                                    <td><b>{{vm.studyAreaService.selectedStudyArea.Pourpoint.Latitude.toUSGSvalue()}}, {{vm.studyAreaService.selectedStudyArea.Pourpoint.Longitude.toUSGSvalue()}}</b></td>
```

Comment:
```suggestion
                                                    <td><b>{{vm.studyAreaService.selectedStudyArea.Pourpoint.Latitude}}, {{vm.studyAreaService.selectedStudyArea.Pourpoint.Longitude}}</b></td>
```

Reply:
Do you mind removing the `Latitude.toUSGSvalue()` and `Longitude.toUSGSvalue()` from the other tabs as well?